lib/plugins/cloud-watch.js

```diff
@@ -15,17 +15,22 @@ function register(skill, cloudwatch, eventMetric) {
     ],
     Namespace: 'StateMachine',
   };
+  const eventMetricConfig = _.merge(defaultEventMetric, eventMetric);
+  const putMetricDataAsync = Promise.promisify(cloudwatch.putMetricData);
+  let positiveEvenMetric = defaultEventMetric;
```

Comment:
i'm not sure what's the difference between eventMetricConfig and positiveEventMetric, you're basically tracking the same thing in `onBeforeReplySent` and `onStateMachineError`